notebooks/data_table_transformer/main.py

```diff
@@ -8,6 +8,7 @@
 
 with herzog.Cell("markdown"):
     """
+    # Data Table Transformer
     ### Use [Pandas](https://pandas.pydata.org) `pivot` command  to convert [AnVIL](https://anvilproject.org)
         sequencing data table imported from Gen3 from long to wide format.
```

Comment:
I think this new line is causing some formatting issues. Maybe also add ### before "sequencing data"?

Reply:
Should be fixed now.